js/functions.js

```diff
@@ -0,0 +1,50 @@
+// Длина строки
+let stringLenth = function (boolean, number) {
+  if (boolean.length <= number) {
+    return true;
+  }
+    return false;
```

Comment:
это лучше заменить на
```suggestion
  return boolean.length <= number
```

Reply:
не исправлено